src/tests/ftest/util/server_utils.py

```diff
@@ -278,16 +315,18 @@ def stop_server(setname=None, hosts=None):
     # reboots
     subprocess.call(["pkill", "^ssh$"])
 
+
 def kill_server(hosts):
-    """
-    Sometimes stop doesn't get everything.  Really whack everything
-    with this.
+    """Force killing the servers.
+
+    Sometimes stop doesn't get everything.  Really whack everything with this.
 
-    hosts -- list of host names where servers are running
+    Args:
+        hosts (list): list of host names where servers are running
     """
     kill_cmds = ["pkill '(daos_server|daos_io_server)' --signal INT",
                  "sleep 5",
                  "pkill '(daos_server|daos_io_server)' --signal KILL"]
     for host in hosts:
```

Comment:
Calling all clush's...

Reply:
Yes, using clush would be better and result in speed bump for all tests, but this should be a separate PR.  I'll create one.